test/integration/isolated/ingress_test.go

```diff
@@ -21,9 +22,12 @@ import (
 
 	"github.com/kong/kubernetes-ingress-controller/v3/internal/annotations"
 	"github.com/kong/kubernetes-ingress-controller/v3/internal/util/builder"
+	incubatorv1alpha1 "github.com/kong/kubernetes-ingress-controller/v3/pkg/apis/incubator/v1alpha1"
```

Comment:
Randomly spotted 🙂  Do we have a rule for `importas` in `.golangci-lint.yaml` to ensure consistent alias `incubatorv1alpha1`? I think we don't have.